src/framework/loader.ts

```diff
@@ -38,6 +44,7 @@ function* concat(lists: IPendingEntry[][]): IterableIterator<IPendingEntry> {
 type TestGroupFilter = (testcase: ICaseID) => boolean;
 function filterTestGroup(group: ITestGroup, filter: TestGroupFilter) {
   return {
+    // What does this syntax mean?
```

Comment:
This is an object-literal definition containing a method, `iterate`, which is a generator (`*`). It's dumb.

Reply:
But the functions filterTestGroup doesn't have a return type?

Reply:
No return type means it's inferred. But there should be a type here (`ITestGroup`); fixed.